app/CsvImporter/Entities/Activity/Components/Elements/ReportingOrganization.php

```diff
@@ -14,17 +14,21 @@
  */
 class ReportingOrganization extends Element
 {
+    /**
+     * @var
+     */
+    public $reportingOrg;
```

Comment:
define a variable type like if it is array or string or integer.

`public $reportingOrg = [];` or `public $reportingOrg = '';`